### PR TITLE
chore: add token logo of Hydranet

### PR DESCRIPTION
**Pull request by Sadhill94**
### 📝 Description

The token is freshly listed on ledger live (thanks for that!), without the icon, we would like to replace that with our brand one.

### ❓ Context

- **Impacted projects**: `live-desktop, @ledgerhq/live-common` 
- **Linked resource(s)**: `` 

Network: Arbitrum
Contract address: 0xF4fe727C855c2D395852ca43F645caB4b504Af23
Explorer URL: https://arbiscan.io/token/0xf4fe727c855c2d395852ca43f645cab4b504af23

### ✅ Checklist

- [ ] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: .changeset/bright-dog-sad.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@ledgerhq/live-common": minor
```
Comment (aelmilligy-ledger) [inline .changeset/bright-dog-sad.md:2]: Hi,

Could you change `minor` to `patch`?

Comment (Sadhill94) [reply .changeset/bright-dog-sad.md:2]: hello, sure!

Comment (Sadhill94) [reply .changeset/bright-dog-sad.md]: updated

**Comment by Sadhill94** [comment]
bump

**Pull request closed without merging**